### PR TITLE
Improved content-loaders for skill search

### LINKED ISSUE #3315: Improve content-loader when skills are searched
**Actual Behaviour**

Currently, when skills are searched in the search bar the same content-loader which is used in the landing page seem to appear.

**Expected Behaviour**

Improve the content loaders.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #3315 

Changes: 
- Improved content-loader to show only when skills are searched.

Demo Link: https://pr-3318-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
**After changes**

Screenshot from 2020-01-21 11-48-22 (1)

Screenshot from 2020-01-21 11-38-05

### CODE REVIEW

**Comment by Dilshaad21** [comment]
@akshatnitd again changes not reflecting in the surge link!

**Comment by Dilshaad21** [comment]
@akshatnitd is this ui fine now?

File: src/components/cms/BrowseSkill/skillLoader.js [hunk 1]
```diff
@@ -47,47 +33,117 @@ const addLoader = () => {
   return loader;
 };
 
-const SkillLoader = () => {
+const skillSearchLoader = () => {
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/skillLoader.js]: Use `SkillSearchLoader`

**Comment by Dilshaad21** [comment]
Will surely make the necessary improvements.

**Pull request merged**